filterizePy/test/test_sharpen.py

```diff
@@ -5,38 +5,52 @@
 
 # February 2019
 # This script tests the sharpen function of filterizePy package.
-
-
 import numpy as np
 import pandas as pd
 import os
 import imghdr
 import skimage.io
-from PIL import Image
-from filterizePy import sharpen
+from scipy.signal import convolve2d
+import matplotlib.pyplot as plt
+from filterizePy.sharpen import sharpen_image
+import pytest
+from keras.preprocessing.image import img_to_array, load_img
+
+
+
+test_path = "img/mirror.png"
 
-def check_input_type(input_img):
+def test_input_type():
     """
     This function checks the input image format is one of the valid image formats.
-
     """
-    assert imghdr.what(input_img) in ['png','jpeg','gif','bmp','jpg'] , "The input image format is incorrect, try another format"
-
+    assert imghdr.what("img/mirror.png") == 'png', "Not a png file, please try again"
```

Comment:
We can try reading in other image types as well!

Reply:
Ok, will revert the change, thank you!